internal/pkg/cli/cmd/template/test.go

```diff
@@ -1,44 +1,54 @@
 package template
 
 import (
-	"fmt"
-
 	"github.com/spf13/cobra"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/dependencies"
 	"github.com/keboola/keboola-as-code/internal/pkg/cli/helpmsg"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/template"
 	testOp "github.com/keboola/keboola-as-code/pkg/lib/operation/template/local/test"
 )
 
 func TestCommand(p dependencies.Provider) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "test <template> [version]",
+		Use:   "test [template] [version]",
 		Short: helpmsg.Read(`template/test/short`),
 		Long:  helpmsg.Read(`template/test/long`),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			d := p.Dependencies()
 
-			if len(args) < 1 {
-				return fmt.Errorf(`please enter argument with the template ID you want to use and optionally its version`)
-			}
-
 			// Get template repository
 			repo, err := d.TemplateRepository(model.TemplateRepository{Type: model.RepositoryTypeWorkingDir, Name: "keboola"}, nil)
 			if err != nil {
 				return err
 			}
 
-			// Optional version argument
-			var versionArg string
-			if len(args) > 1 {
-				versionArg = args[1]
-			}
-
-			// Load template
-			template, err := d.Template(model.NewTemplateRef(repo.Ref(), args[0], versionArg))
-			if err != nil {
-				return err
+			// Load templates
+			templates := make([]*template.Template, 0)
+			if len(args) >= 1 {
+				// Optional version argument
+				var versionArg string
+				if len(args) > 1 {
+					versionArg = args[1]
+				}
+				tmpl, err := d.Template(model.NewTemplateRef(repo.Ref(), args[0], versionArg))
+				if err != nil {
+					return err
+				}
+				templates = append(templates, tmpl)
+			} else {
```

Comment:
Když se nezadá templata v argumentu, načtou se všechny. Přidal jsem na to e2e test.